src/components/AppProvider/utilities/createAppProviderContext/createAppProviderContext.ts

```diff
@@ -27,14 +26,13 @@ export default function createAppProviderContext({
 }: CreateAppProviderContext = {}): Context {
   const intl = new Intl(i18n);
   const link = new Link(linkComponent);
-  const appBridge =
-    apiKey && !isServer
-      ? createApp({
-          apiKey,
-          shopOrigin: shopOrigin || getShopOrigin(),
-          forceRedirect,
-        })
-      : undefined;
+  const appBridge = apiKey
+    ? createApp({
+        apiKey,
+        shopOrigin: shopOrigin || getShopOrigin(),
```

Comment:
is this change intended for the current version of polaris or the next major release?

Reply:
This is intended to be a bug fix for the current version of Polaris.

Reply:
👍